src/data/faq_de.json

```diff
@@ -580,9 +580,9 @@
                         "anchor": "new_device",
                         "active": true,
                         "textblock": [
-                            "Ein Backup der alten Zufalls-IDs ist leider nicht möglich, da die Exposure Notification API von Apple/Google keinen Export dieser Daten erlaubt. Damit Sie trotzdem über mögliche Risikobegegnungen informiert werden, empfehlen wir Ihnen, den Risikostatus auf dem alten Gerät noch mindestens 14 Tage zu beobachten, nachdem Sie die Corona-Warn-App auf dem neuen Gerät installiert haben. Beachten Sie dazu folgende Hinweise:",
+                            "Ein Backup der alten Zufalls-IDs ist leider nicht möglich, da die Exposure Notification Framework von Apple/Google keinen Export dieser Daten erlaubt. Damit Sie trotzdem über mögliche Risikobegegnungen informiert werden, empfehlen wir Ihnen, den Risikostatus auf dem alten Gerät noch mindestens 14 Tage zu beobachten, nachdem Sie die Corona-Warn-App auf dem neuen Gerät installiert haben. Beachten Sie dazu folgende Hinweise:",
                             "<ul><li>Ihr altes Gerät muss in der Lage sein, regelmäßige Überprüfungen auf mögliche Begegnungen durchzuführen. Lassen Sie das Gerät daher permanent eingeschaltet und dauerhaft mit dem Internet verbunden.</li><li>Bewahren Sie das alte Gerät an einem stationären Ort auf, z.B. an einem bestimmten Platz in Ihrer Wohnung. Falls möglich, platzieren Sie das Gerät in Hör- oder Sichtweite, sodass Sie bei einer Benachrichtigung der App frühzeitig gewarnt werden.</li><li>Rufen Sie die Corona-Warn-App auf dem alten Gerät mindestens einmal täglich aktiv auf und überprüfen Sie dort Ihren Risikostatus.</li></ul>",

```

Comment:
Perhaps instead of 
"in Ihrer Wohnung"
use
"zu Hause"?
since this is more general.